GNUmakefile

```diff
@@ -102,6 +102,15 @@ presubmit: ## Format go files and imports.
 mockery: $(mockery) ## Install mockery.
 	go install github.com/vektra/mockery/v2@v2.20.0
 
+.PHONY: codecgen
+codecgen: $(codecgen)
+	go install github.com/ugorji/go/codec/codecgen@v1.2.10
+
+.PHONY: run-codecgen
+run-codecgen: $(run-codecgen)
+	cd core/chains/evm/types/internal/blocks && codecgen -o block_codecgen.go  -j true block.go && codecgen -o transactions_codecgen.go  -j true transactions.go 
```

Comment:
We normally do this sort of thing (e.g. mockery) from `//go:generate` commands inside the package/files.

Reply:
done